docs/_includes/helpers/embed.html

```diff
@@ -0,0 +1,34 @@
+{% assign include_path = include.embed | prepend: '_embeds/' | append: '.md' %}
+{% for embed in site.embeds %}
+  {% if embed.path == include_path %}
+    {% assign embed_code = embed.content %}
+    {% assign embed_title = embed.title %}
+  {% endif %}
+{% endfor %}
+
+<iframe
+  title="{{ embed_title | default: 'Component Preview' }}"
+  src="{{ site.baseurl }}/embeds/{{ include.embed }}"
+  width="100%"
+  height="400"
+  frameborder="0"
+  class="margin-y-3"
+></iframe>
+<script>
+  /** @type {HTMLIFrameElement} */
+  const iframe = document.currentScript.previousElementSibling;
```

Comment:
Now that I think about it, since we don't actually sandbox the frame and it's hosted on the same domain, we could probably just add an observer directly from this point:

```suggestion
  const iframe = document.currentScript.previousElementSibling;
  const { body: frameBody } = iframe.contentWindow.document;
  new ResizeObserver(() => {
    iframe.height = frameBody.offsetHeight;
  }).observe(frameBody);
```

Then again, maybe we do want to sandbox, or at least treat as if it's sandboxed? I don't know that it's a whole lot more complex as implemented.

Reply:
do we *need* resize support? this is local previews only, could we have folks just hit refresh after they resize?

Reply:
We don't strictly need it, but we also can't know the natural height of the preview ahead-of-time†, so if we didn't have this, we'd have to set the height to something static and let the page either (a) be taller than necessary or (b) have scroll.

† Technically, we could measure this ahead of time, either manually or through some script using something like Puppeteer to measure the heights of each rendered preview, but that would be far more involved than what this implementation is currently doing.